iOS-sample-app/Showcases/DynamicVariables/DynamicVariablesViewController.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+import ContentSquare
+
+enum AccountType: Int {
+    case standard = 0
+    case premium = 1
+}
+
+// This class showcases how to use dynamic variables to gather additional data about the session,
+// such as which version of an A/B test the user saw, or other relevant user data
+class DynamicVariablesViewController: UIViewController {
+
+    @IBOutlet private weak var testButton: UIButton!
+    private var testButtonColor = UIColor()
+    private var userAge: Int = 0
+    private var userAccount: AccountType = .standard
+
+    override func loadView()
+    {
+        super.loadView()
+
+        // Get the color for the button from A/B test configuration
+        // For instance, yellow or green
+        testButtonColor = .yellow
+        testButton.backgroundColor = testButtonColor
+        // Retrieve the user's age and his account type, from his personal account for instance
+        userAge = 24
```

Comment:
I'd have set userAge directly in the declaration of the variable, with a comment about this value is retrieved from an external source or something like that (actually, I'd even go a bit further and create a different "Account" class with age and account type as properties, have an instance of it defined as a static singleton outside of DynamicVariablesViewController, just for the purpose of faking it and accessing it in a more credible way.

Reply:
I think it's a bit overkill: the purpose is to demonstrate how to use the DynamicVar, with examples of meaningful keys/values, we don't need to show users how to implement a user account.